docs/getting_started.rst

```diff
@@ -0,0 +1,361 @@
+
+2.1. **Log in**
+
+  .. note:: This section is for those who have an online server. If you have
+    a local Linux machine, just open the terminal and skip to the next part.
+
+  As we said before, a VPS is controlled through command line. You will have to
+  connect to your VPS through a protocol called SSH.
+
+  .. image:: .resources/instances-ssh-button.png
+
+  On your host page (here, it is Google Cloud), find the SSH button and click on
+  it. You will be connected to your server with command line. You should see
+  something like this.
+
+  .. image:: .resources/ssh-output.png
+
+  .. note:: Don't forget to type the command ``logout`` to close the SSH properly.
+
+3. **Install and set up Red**
+
+  Just follow one of the Linux installation guide. We provide guides for the
+  most used distributions. Check the :ref:`home page <main>` and search for
+  your distribution.
+
+4. **Set up an auto-restart**
+
+  Once you got Red running on your server, it will still shut down if you close
+  the window. You can set up an auto-restarting system that will create a
+  side task and handle fatal errors, so you can just leave your server running
+  and enjoy Red!
+
+  For that, just follow :ref:`this guide <systemd-service-guide>`.
+
+.. _getting-started-userdocs:
+
+------------------
+User documentation
+------------------
+
+You will soon start using the Red core cogs. A detailled documentation is
+available for every core cog, under the :ref:`How to use <main>` section.
+
+The cog guides are formatted the same. They're divided into 3 sections:
+
+* **Guide**
+
+  This will introduct you to the cog and explain you how it works.
+
+* **Commands**
+
+  A list of the available commands, with details and arguments.
+  Each command guide will be formatted like this:
+
+  * **Syntax**
+
+    A line that will show how the command must be invoked, with the arguments.
+
+    .. tip:: If the command show something like ``[lavalinkset|llset]``, that means
+        you can invoke the command with ``lavalinkset`` or with ``llset``, this is
+        called an alias.
+
+  * **Description**
+
+    A detailled description of what the command does, with details about how
+    it must be used.
+
+  * **Arguments**
+
+    A list of all arguments needed (or not) for the command, with more details.
+
+    .. tip::
+        Arguments enclosed in ``< >`` means that the argument is **necessary**
+        for the command to work.
+
+        Arguments enclosed in ``[ ]`` means that the command is **optional**
+        for the command; you can decide to use it or not.
+
+        Arguments followed by ``=something`` means that, if not specified,
+        the argument will be equal to ``something``.
+
+        For example, ``[days=1]`` in the ``ban`` command means that the number
+        of days of messages to be deleted will be equal to ``1`` if not
+        specified.
+
+* **Frequently asked questions**
```

Comment:
Why add this if nothing is there?

Reply:
Was supposed to be used at first, forgot to remove it.